pages/collective-page.js

```diff
@@ -178,6 +180,10 @@ class CollectivePage extends React.Component {
 
     const collective = data && data.Collective;
 
+    if (mode === 'apply' && !collective.isHost) {
+      return <Custom404 />;
+    }
+
```

Comment:
Why is this needed? Code comment perhaps?

Reply:
As we don't want `/webpack/apply` to be a valid URL. I will add a comment.